packages/move/sources/vesting.move

```diff
@@ -138,6 +177,8 @@ module deployment_addr::vesting {
         let sender_addr = signer::address_of(sender);
         let config = borrow_global<Config>(@deployment_addr);
         assert!(config.admin == sender_addr, ERR_ONLY_ADMIN_CAN_ADD_VESTING_STREAM);
+
+        // FIX: what's the concern with using >= instead of > below?
```

Comment:
I think its a matter of preference. My assumption was that the start_time should be in the future, so I used > instead of >=.